### PR TITLE
feat: run docker container as non root user with permissions

### LINKED ISSUE #6505: Run docker container as non root user
Currently, the app and celery run as root in docker containers 

They should be run as normal users

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #6505 

`WIP`

#### Short description of what this resolves:
feat: run docker container as non root user with permissions

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by codedsun** [comment]
@iamareebjamal Test please

File: Dockerfile [hunk 1]
```diff
@@ -25,5 +25,13 @@ RUN apk --no-cache add postgresql-libs ca-certificates libxslt jpeg zlib file li
 WORKDIR /data/app
 ADD . .
 
+RUN adduser -S open_event_user
+# Gives permission to open_event_user
+RUN chown -R open_event_user /data/app
+RUN chown -R open_event_user ./static
+RUN chown -R open_event_user ./generated
```
Comment (iamareebjamal) [inline Dockerfile]: This is run at creation of the docker image, so it ran on your machine only. It'll never run on server

Comment (codedsun) [reply Dockerfile]: Then to run on the server, what should be done? Yes while building the image this was done. But in tomorrow PR how does it work?

Comment (iamareebjamal) [reply Dockerfile]: Because that folder is inside the container and these are on host

Comment (codedsun) [reply Dockerfile]: @iamareebjamal - Okay but can you show me the direction about how to solve this now. We want this to run without building the image.?

Comment (codedsun) [reply Dockerfile]: Means what exactly the issue is here. We wont build the image again on server and I have to add these host files permission which are outside the container.

Comment (iamareebjamal) [reply Dockerfile]: Searching for better solution

**Pull request closed without merging**